ds/addr_book.go

```diff
@@ -185,7 +193,7 @@ func (mgr *DatastoreAddrManager) Addrs(p peer.ID) []ma.Multiaddr {
 }
 
 // Peers returns all of the peer IDs for which the AddrBook has addresses.
-func (mgr *DatastoreAddrManager) Peers() []peer.ID {
+func (mgr *dsAddrBook) AddrsPeers() []peer.ID {
```

Comment:
Had to change method names to avoid collisions when the `KeyBook` and `AddrBook` interfaces are embedded in `Peerstore`. Added a `Peers()` method [in the `Peerstore` interface](https://github.com/libp2p/go-libp2p-peerstore/pull/30/files#diff-e2cd38f38509c9033189fbf0317e414eR69) to keep backwards compatibility.